Week 3/NomadBusinessCard/app/src/main/res/layout/switch_item.xml

```diff
@@ -0,0 +1,11 @@
+<RelativeLayout
```

Comment:
I'm not sure what this layout is for, I can't find it again throughout the project.